docs/snap-clone.md

```diff
@@ -42,9 +42,9 @@
     ./scripts/install-snapshot.sh cleanup
     ```
 
-**NOTE: At present, there is a limit of 400 snapshots per cephFS filesystem.
-Also PVC cannot be deleted if it's having snapshots. Make sure all the snapshots
-on the PVC are deleted before you delete the PVC.**
+**NOTE: A PVC cannot be deleted if it's having snapshots.
+Make sure all the snapshots on the PVC are deleted
+before you delete the PVC.**
```

Comment:
this is not true anymore, we do allow PVC deletion if its having snapshots as well.